assembl/views/__init__.py

```diff
@@ -86,7 +96,8 @@ def find_theme(theme_name, frontend_version=1):
             (head, name) = os.path.split(dirpath)
             print name, relpath
             if name == theme_name:
-                return relpath
+                THEMES[theme_name] = relpath
+                return THEMES[theme_name]
```

Comment:
You need two caches based on frontend_version. THEMES[frontend_version][theme_name]

Reply:
You can `return relpath` instead of accessing the dict.